.travis.yml

```diff
@@ -2,27 +2,55 @@ sudo: false
 
 language: php
 
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - vendor
+
+env:
+  global:
+    - COMPOSER_ARGS="--no-interaction --ignore-platform-reqs --no-scripts"
+
+notifications:
+  irc: "irc.freenode.org#zftalk.dev"
+  email: false
+
 matrix:
   fast_finish: true
   include:
-    - php: 5.5
```

Comment:
If we do this for master, we need to keep 5.5 in here; if we do it for develop, and the 1.1 release, we can leave as-is. I'm going to leave it to you, @xtreamwayz , to decide which you want to target.